pkg/features/client_adapter.go

```diff
@@ -65,7 +65,7 @@ func (a *clientAdapter) Add(in map[clientfeatures.Feature]clientfeatures.Feature
 		}
 		out[featuregate.Feature(name)] = converted
 	}
-	return a.mfg.Add(out)
+	return a.mfg.Add(out) //nolint:forbidigo
```

Comment:
When I see a `//nolint`, I prefer to have some explanation of what is suppressed and why. In this case, I suppose it has to be this way? Is this proposal correct?

```suggestion
	return a.mfg.Add(out) //nolint:forbidigo // Must use Add without support for versioned feature gate here.
```

Reply:
Added the nolint reason for all the exceptions.